### PR TITLE
mentors can see the most recent results

### PR DESCRIPTION
**Pull request by naderakhgari**
Thank you for contributing! To help the review process, please provide the following:

### Proposal
mentors can see the most recent results

### Related
Provide links to any associated pull requests or issues. You can use `#123` to link to a PR or issue in this repository, or `user/repo#123` to link to other repositories.

### Checklist

- [ ] I have read the [code of conduct] and [contributing guide]
- [ ] I have made this pull request to the `master` branch
- [ ] I have run all of the automated validation using `npm run ship`
- [ ] I have added myself to the `"contributors"` list in the `package.json` (or do not want to)

[code of conduct]: https://github.com/textbook/starter-kit/blob/master/.github/CODE_OF_CONDUCT.md
[contributing guide]: https://github.com/textbook/starter-kit/blob/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: client/src/Components/Results.js [hunk 1]
```diff
@@ -1,154 +1,191 @@
...
+			);
 		}
 	};
-	useEffect(() => {
-		if(allResults){
-			makeNames();
+
+	const colorState = (correct) => {
+		if (correct) {
+			return "corect";
```
Comment (pendletons) [inline client/src/Components/Results.js:112]:
```suggestion
			return "correct";
```

File: client/src/Components/Results.js [hunk 2]
```diff
@@ -1,154 +1,191 @@
...
+		} else {
+			return "incorect";
```
Comment (pendletons) [inline client/src/Components/Results.js:114]:
```suggestion
			return "incorrect";
```

File: client/src/Components/Results.js [hunk 3]
```diff
@@ -1,154 +1,191 @@
...
-	},[quizSelected]);
-	const makeNames=()=>{
-		let tempNames = [];
-		if(allResults!=="not found!"){
-			allResults.map((oneAnswer)=>{
-				if(studentNames&&!tempNames.includes(oneAnswer.studentName)){
-					tempNames.push(oneAnswer.studentName);
+
+	const getValues = (student, questionId) => {
+		const allatemmpts = quizSelectedResult.filter(
```
Comment (pendletons) [inline client/src/Components/Results.js:70]: `allatemmpts` should be `allAttempts`

**Review by pendletons — CHANGES_REQUESTED** [review]
Looks good - I commented on a few typos.

**Review by cockochan — APPROVED** [review]
need to merge before demo

**Pull request merged**